src/timeMachine/utils/insertFunction.ts

```diff
@@ -24,3 +24,110 @@ export const generateImport = (
 
   return importStatement
 }
+
```

Comment:
I'll remove comments once pr gets approved. Just wanted to provide context for now :)